src/styles/Login.scss

```diff
@@ -1,17 +1,25 @@
-.login-container {
+// TODO: white theme the colors look ugly - fix
+
+.login-container-container {
```

Comment:
Please fix all the css bugs here, the login page is broken on smaller devices. You can make a compromise between sexy and alright on all devices, but this is not it.